packet/endpoints.go

```diff
@@ -32,6 +33,47 @@ type ComponentsResponse struct {
 	Components []Component `json:"components"`
 }
 
+// GetWorkflowsFromTink fetches the list of workflows from tink
+func (c *Client) GetWorkflowsFromTink(hwID string) (tw.WorkflowContextList, error) {
+
+	var result tw.WorkflowContextList
+
+	if hwID == "" {
+		return result, errors.New("missing hardware id")
+	}
+
+	labels := prometheus.Labels{"from": "dhcp"}
+	cacherTimer := prometheus.NewTimer(metrics.CacherDuration.With(labels))
+	metrics.CacherRequestsInProgress.With(labels).Inc()
+	metrics.CacherTotal.With(labels).Inc()
+
+	if c.workflowClient == nil {
+		return result, errors.New("Unable to fetch workflow, client does not exist")
```

Comment:
Unable should be lower case `u`

Reply:
done

Reply:
i found it  while addressing review comments, have fixed it